kbase.yml

```diff
@@ -8,7 +8,7 @@ service-language:
     python
 
 module-version:
-    0.4.3
+    0.4.6
```

Comment:
why 0.4.3 to 0.4.6

Also maybe this is an opportunity to add a changelog? I like to use https://keepachangelog.com/en/1.0.0/

Reply:
I think it had something to do with catalog registration not working